### PR TITLE
Store `raw_request` in `ModelInference` ClickHouse table and remove `input` and `output` columns

### PR DESCRIPTION
**Pull request by virajmehta**

### CODE REVIEW

File: gateway/tests/e2e/providers/aws_bedrock.rs [hunk 1]
```diff
@@ -133,22 +133,12 @@ async fn test_inference_with_explicit_region() {
...
-    let content_blocks: Vec<Value> = serde_json::from_str(output).unwrap();
-    assert_eq!(content_blocks.len(), 1);
-    let content_block = content_blocks.first().unwrap();
-    let content_block_type = content_block.get("type").unwrap().as_str().unwrap();
-    assert_eq!(content_block_type, "text");
-    let clickhouse_content = content_block.get("text").unwrap().as_str().unwrap();
-    assert_eq!(clickhouse_content, content);
-
+    let raw_request = result.get("raw_request").unwrap().as_str().unwrap();
+    assert!(raw_request.to_lowercase().contains("world!"));
```
Comment (GabrielBianconi) [inline gateway/tests/e2e/providers/aws_bedrock.rs:141]: All these checks should confirm it's valid JSON (esp. common.rs)

**Review by GabrielBianconi — CHANGES_REQUESTED** [review]
Repeat my comments across all providers and tests

File: gateway/src/inference/providers/anthropic.rs [hunk 2]
```diff
@@ -533,16 +542,26 @@ struct AnthropicResponse {
     usage: AnthropicUsage,
 }
 
-#[derive(Clone, Debug, PartialEq, Serialize)]
-struct AnthropicResponseWithLatency {
+#[derive(Debug, PartialEq, Serialize)]
+struct AnthropicResponseWithMetadata<'a> {
     response: AnthropicResponse,
     latency: Latency,
+    request_body: AnthropicRequestBody<'a>,
```
Comment (GabrielBianconi) [inline gateway/src/inference/providers/anthropic.rs:549]: We could call this just `request` since the `response` is also the body.

**Review by GabrielBianconi — APPROVED** [review]

File: gateway/src/inference/providers/anthropic.rs (hunk 2 above)
Comment (virajmehta) [reply gateway/src/inference/providers/anthropic.rs:549]: kk

**Pull request merged**